### ISSUE TITLE
Add support for other languages

### ISSUE THREAD

**Issue by GeorgeCiesinski** [GeorgeCiesinski/poke-guesser-bot#5]
# Describe the new feature
The bot currently only accepts English answers. Functionality should be added to take advantage of pokeApi's multi language support so that pokemon names can be guessed in any supported language.

### fetchNames-Implementation

```
function fetchNames(nameOrId) {
  return fetch('https://pokeapi.co/api/v2/pokemon-species/' + nameOrId + '/')
  // Parse to json
  .then(res => res.json())
  // Get names as array
  .then(json => json.names)
  // Format names
  .then(names => {
    let resultNames = []
    for (let i = 0; i < names.length; i++) {
      resultNames.push({
        languageName: names[i].language.name,
        languageUrl: names[i].language.url,
        name: names[i].name
      })
    }
    return resultNames
  })
}
```

### Guess-Check

```
pokeFetch.fetchNames(pokemonArray[0].toLowerCase()).then(names => {
        console.log('Guess:' + guess)
        for (let i = 0; i < names.length; i++) {
          if (names[i].name.toLowerCase() === guess.toLowerCase()) {
            addScore(msg.author.username)
            // Send message that guess is correct
            msg.channel.send(`${msg.author} correctly guessed ${guess}`)
            break; // To avoid scoring multiple times
          }
        }
      })
```

## Useful links

1. [Dominik's example](https://replit.com/@Wissididom/Pokemon-Discord-Bot-Testing)

**Comment by Wissididom** [comment]
The addScore here should also be updated like mentioned in #6 when we switch over to ids. Just so that we don't forget.

**Comment by GeorgeCiesinski** [comment]
Fixed in Pull Request #10 

As the addScore change has been added to #6 I will proceed to close this issue.
